Pagination: Add protection for empty payloads

Currently it just returns an unhandled error:
```
|8/28/2023, 10:08:33 PM| [EE] core: An unhandled async error occurred: ```
TypeError
TypeError: Cannot read properties of undefined (reading 'components')
    at PaginatedMessage.renderCurrentPage (/home/meti/Documents/TuringBot/src/core/pagination.ts:162:17)
    at PaginatedMessage.replyAndRegisterListener (/home/meti/Documents/TuringBot/src/core/pagination.ts:185:24)
    at PaginatedMessage (/home/meti/Documents/TuringBot/src/core/pagination.ts:94:10)
    at SubModule.executeCommand (/home/meti/Documents/TuringBot/src/modules/duck/duck.ts:718:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)```
^Cmake: *** [Makefile:22: start] Interrupt
```

--- a/src/core/pagination.ts
+++ b/src/core/pagination.ts
@@ -137,7 +137,6 @@ export class PaginatedMessage {
     const paginationRow: ActionRowBuilder<ButtonBuilder> =
       this.generateControlRow() as ActionRowBuilder<ButtonBuilder>;
 
-    
     // If payloads is null or empty, someone provided incorrect arguments to this class, return early
     if (this.payloads === null || this.payloads.length === 0) {
       return {
@@ -255,7 +254,7 @@ export class PaginatedMessage {
       if (this.payloads === null || this.payloads.length === 0) {
         return;
       }
-      
+
       // If the interaction is supposed to be deleted afterwards and wasn't stopped manually, delete it
       if (this.deleteAfter && !this.stoppedManually) {
         await interaction.deleteReply();

